examples/googlecalendar/views/index.mustache

```diff
@@ -8,7 +8,7 @@
         <div class="card text-center" style="width: 24rem;">
             <div class="card-body bg-light">
               <h5 class="card-title">Google OAuth Calendar</h5>
-              <p class="card-text">A simple OAuth2 Application that will authenticate and authorize to access your Google Calendar and display the events.</p>
+              <p class="card-text">A simple OAuth2 Application that will authenticate and authorize to access your Google Calendar and display events.</p>
```

Comment:
`A simple OAuth2 Application that will access your Google Calendar and display events.`